src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,39 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) {
+    if (typeof obj !== "object") {
```

Comment:
You'd be surprised how many things pass this check. `Date`, `Array`, etc. This isn't exactly a comprehensive check.